backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/scheduling/policyunits/NumaPolicyUnit.java

```diff
@@ -84,6 +85,11 @@ public List<VDS> filter(SchedulingContext context, List<VDS> hosts, List<VM> vmG
                 continue;
             }
 
+            // Clear resize vm on run command.
+            vmsToCheckOnHost = vmsToCheckOnHost.stream()
```

Comment:
how about having a variable named like vmsWithNumaNodes instead of reusing vmsToCheckOnHost and drop the comment above? I agree that the comment is needed with the posted code because it's not that clear why we change vmsToCheckOnHost but if we'll use a variable with a better name, we won't need it